cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -0,0 +1,71 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import six
+
+if six.PY3:
+    from typing import Optional
+
+    from pydantic import BaseModel, Field, computed_field
+
+    class NodeAttributes(BaseModel):
+        address: Optional[str] = None
+        ad_st: Optional[str] = Field(default=None, alias="adSt")
+        role: Optional[str] = None
+        dn: Optional[str] = None
+        model: Optional[str] = None
+        version: Optional[str] = None
+        serial: Optional[str] = None
+        vendor: Optional[str] = Field(default='cisco_aci')
+        namespace: Optional[str] = Field(default='default')
+
+    class Node(BaseModel):
+        attributes: NodeAttributes
+
+    class EthAttributes(BaseModel):
+        admin_st: Optional[str] = Field(default=None, alias="adminSt")
+        id: Optional[str] = None
+        name: Optional[str] = None
+        desc: Optional[str] = None
+        router_mac: Optional[str] = Field(default=None, alias="routerMac")
+
+    class Eth(BaseModel):
+        attributes: EthAttributes
+
+    class DeviceMetadata(BaseModel):
+        device_id: Optional[str] = Field(default=None)
+        id_tags: list = Field(default_factory=list)
+        tags: list = Field(default_factory=list)
+        name: Optional[str] = Field(default=None)
+        ip_address: Optional[str] = Field(default=None)
+        model: Optional[str] = Field(default=None)
+        ad_st: Optional[str] = Field(default=None, exclude=True)
+        vendor: Optional[str] = Field(default=None)
+        version: Optional[str] = Field(default=None)
+        serial_number: Optional[str] = Field(default=None)
+
+        @computed_field
+        @property
+        def status(self) -> int:
+            return 1 if self.ad_st == 'on' else 2
+
+    class DeviceMetadataList(BaseModel):
+        device_metadata: list = Field(default_factory=list)
+
+    class InterfaceMetadata(BaseModel):
+        device_id: Optional[str] = Field(default=None)
+        id_tags: list = Field(default_factory=list)
+        index: Optional[str] = Field(default=None)
+        name: Optional[str] = Field(default=None)
+        description: Optional[str] = Field(default=None)
+        mac_address: Optional[str] = Field(default=None)
+        admin_status: Optional[str] = Field(default=None, exclude=True)
+
+        @computed_field
+        @property
+        def status(self) -> int:
```

Comment:
Similarly, this is the admin_status property of the interface.
There seems to also be a "status" field on the interface, but examples I found online have an empty string in this place.
Have you tried the API call on the sandbox, is there anything we can get?

I don't think `admin_status` should be mapped to the status

Reply:
yes, woops i realize there's an `operStatus` that will be the operational status which i think is probably a better representation? there are a million statuses 😓 my bad - good catch! i'll update the API call so that it gets the additional stat for each interface + parse it so it can be added to the call to EvP

Reply:
also realized belatedly i don't need to set status for interfaces, just pass the admin/oper status woopsie